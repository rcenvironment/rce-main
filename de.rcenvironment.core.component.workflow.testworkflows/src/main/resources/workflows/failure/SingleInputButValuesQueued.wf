{
  "identifier" : "1bddadb7-7a08-4405-8403-626a1944e8fe",
  "workflowVersion" : "4",
  "nodes" : [ {
    "identifier" : "2faa322a-0bb4-41fc-ad06-924c5af95ba2",
    "name" : "Parametric Study",
    "location" : "149:66",
    "active" : "true",
    "component" : {
      "identifier" : "de.rcenvironment.parametricstudy",
      "name" : "Parametric Study",
      "version" : "3.3"
    },
    "staticOutputs" : [ {
      "identifier" : "c97116ae-fa3c-4d80-9b8e-123ca0312be6",
      "name" : "Design variable",
      "datatype" : "Float",
      "metadata" : {
        "FromValue" : "0",
        "StepSize" : "1",
        "ToValue" : "10"
      }
    } ]
  }, {
    "identifier" : "6b9ae8ed-3c65-4cfd-a541-1a5b0f1c6f8c",
    "name" : "Single",
    "location" : "285:66",
    "active" : "true",
    "component" : {
      "identifier" : "de.rcenvironment.script",
      "version" : "3.4",
      "name" : "Script"
    },
    "configuration" : {
      "pythonExecutionPath" : "${pythonExecutionPath}",
      "usageOfScript" : "NEW",
      "storeComponentHistoryData" : "false",
      "script" : "RCE.read_input('x')",
      "scriptLanguage" : "Jython"
    },
    "dynamicInputs" : [ {
      "identifier" : "0c143f8c-a9fa-46a8-81b3-108b40f476ca",
      "name" : "x",
      "epIdentifier" : "default",
      "datatype" : "Float",
      "metadata" : {
        "inputExecutionConstraint_4aae3eea" : "Required",
        "inputHandling_73b1056e" : "Single"
      }
    } ]
  } ],
  "connections" : [ {
    "source" : "2faa322a-0bb4-41fc-ad06-924c5af95ba2",
    "output" : "c97116ae-fa3c-4d80-9b8e-123ca0312be6",
    "target" : "6b9ae8ed-3c65-4cfd-a541-1a5b0f1c6f8c",
    "input" : "0c143f8c-a9fa-46a8-81b3-108b40f476ca"
  } ]
}