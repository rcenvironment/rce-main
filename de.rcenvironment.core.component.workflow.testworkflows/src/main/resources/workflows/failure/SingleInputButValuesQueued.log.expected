A new value at input 'x' of type 'single' received, but the current one was not consumed yet